package.json

```diff
@@ -33,29 +34,18 @@
 	},
 	"homepage": "https://discord.js.org",
 	"engines": {
-		"node": ">=16.9.0"
+		"node": ">=18"
 	},
-	"workspaces": [
-		"_base",
-		"basic",
-		"radio-bot",
-		"recorder"
-	],
-	"packageManager": "npm@8.11.0",
+	"packageManager": "pnpm@9.1.4",
```

Comment:
Can we also get volta pinnings in here? 🙏 you can probably copy it from other repos of mine if need be, but this isn't a blocker